cloudops.go

```diff
@@ -48,6 +48,9 @@ type Compute interface {
 	// InspectInstanceGroupForInstance inspects the instance group to which the
 	// cloud instance with given ID belongs
 	InspectInstanceGroupForInstance(instanceID string) (*InstanceGroupInfo, error)
+	// SetCountForInstanceGroup sets desired count per availability zone
+	// for given instance group
+	SetCountForInstanceGroup(instanceGroupInfo *InstanceGroupInfo, count int64) error
```

Comment:
Don't take the whole instance group info. Have the API specifically take in only what's needed

Reply:
@harsh-px updated this API to just accept `instanceID` as input.
Also refactored both `InspectInstanceGroupForInstance ()` and `SetCountForInstanceGroup ()` to use common helper function `getGkeClusterDetails()`.